cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -19,14 +19,16 @@ import(
  "context"
  "flag"
  "fmt"
+ "io"
+ "io/ioutil"
  "log"
 
 foo "path.to/client/foo"
 foopb "path.to/pb/foo"
 )
 // [START awesome_region]
 
-func sampleUnaryMethod(theX int64, theB string, theFoo string) error {
+func sampleUnaryMethod(theX int64, theB string, theFoo string, fileArg string) error {
```

Comment:
What is the YAML equivalent of the definition of this function?

I'm having a hard time reviewing it without clear YAML inputs

Reply:
Here's the yaml:
```
- parameters:
    defaults:
    - file_arg = "path/to/local/file/arg.txt"
    - file_default = "path/to/local/file/default.txt"
    attributes:
    - parameter: file_arg:
        sample_argument_name: file_arg
        read_file: true
    - parameter: file_default:
        read_file: true
```